store/sdk/src/main/java/org/apache/carbondata/sdk/file/CarbonReaderBuilder.java

```diff
@@ -228,4 +228,118 @@ public CarbonReaderBuilder withRowRecordReader() {
     }
   }
 
+  /**
+   * gets an array of blocklet level CarbonInputSplits
+   * @return
+   * @throws IOException
+   */
+  public InputSplit[] getSplits() throws IOException {
+    if (hadoopConf == null) {
+      hadoopConf = FileFactory.getConfiguration();
+    }
+    CarbonTable table;
+    table = CarbonTable.buildTable(tablePath, tableName, hadoopConf);
+    final CarbonFileInputFormat format = new CarbonFileInputFormat();
+    final Job job = new Job(hadoopConf);
```

Comment:
Please use new method. this method is deprecated.

Reply:
ok